nvflare/dashboard/application/project.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from flask import current_app as app
+from flask import jsonify, make_response, request
+from flask_jwt_extended import create_access_token, get_jwt, jwt_required
+
+from . import jwt
+from .store import Store
+
+
+@jwt.expired_token_loader
+def my_expired_token_callback(jwt_header, jwt_payload):
+    return jsonify({"status": "unauthenticated"}), 401
+
+
+@app.route("/api/v1/login", methods=["POST"])
+def login():
+    req = request.json
+    email = req.get("email", None)
+    password = req.get("password", None)
+    user = Store.verify_user(email, password)
+    if user:
+        additional_claims = {"role": user.role.name, "organization": user.organization.name}
+        access_token = create_access_token(identity=user.email, additional_claims=additional_claims)
+        return jsonify(
+            {
+                "status": "ok",
+                "user": {"id": user.id, "email": user.email, "role": user.role.name},
+                "access_token": access_token,
+            }
+        )
+    else:
+        return jsonify({"status": "unauthenticated"}), 401
+
+
+@app.route("/api/v1/dummy", methods=["GET"])
+def dummy_blob():
+    fileobj = Store.dummy()
+    return jsonify({"status": "ok"})
```

Comment:
do we need this api?

Reply:
That's for testing and should be removed.